pkg/storage/fs.go

```diff
@@ -0,0 +1,25 @@
+package storage
```

Comment:
@skyf0cker can we move the content of this file to `pkg/storage/local.go`? :)

Reply:
Have changed it. Thanks for your review :)